package.json

```diff
@@ -24,10 +24,14 @@
    },
    "homepage": "https://github.com/anssi55/Epic-bulletin-board-backend-project#readme",
    "dependencies": {
+      "@types/chai-http": "^4.2.0",
       "@types/express": "^4.17.0",
+      "@types/mocha": "^5.2.7",
```

Comment:
All types packages should be in devDependencies. When installing stuff to devDependencies you can do it like this `npm i -D @types/mocha`